### PR TITLE
tests: bump dd-trace-go

### PR DESCRIPTION
**Pull request by jirikuncar**
The new version of `dd-trace-go` is sending `os.*` and `runtime.*` tags for detailed test information.

### CODE REVIEW

File: LICENSE-3rdparty.csv [hunk 1]
```diff
@@ -54,3 +54,7 @@ go.sum,github.com/chzyer/readline,MIT,2015 Chzyer
 go.sum,github.com/chzyer/test,MIT,2016 Chzyer
 go.sum,github.com/google/pprof,Apache-2.0,Google Inc.
 go.sum,github.com/ianlancetaylor/demangle,BSD-3-Clause,2015 The Go Authors
+
+go.sum,github.com/Microsoft/go-winio,<LICENSE>,<COPYRIGHT>
+go.sum,github.com/konsorten/go-windows-terminal-sequences,<LICENSE>,<COPYRIGHT>
+go.sum,github.com/sirupsen/logrus,<LICENSE>,<COPYRIGHT>
```
Comment (zippolyte) [inline LICENSE-3rdparty.csv:60]: Template values should be filled

**Review by zippolyte — CHANGES_REQUESTED** [review]

File: LICENSE-3rdparty.csv (hunk 1 above)
Comment (jirikuncar) [inline LICENSE-3rdparty.csv:60]:
```suggestion
go.sum,github.com/Microsoft/go-winio,MIT,2015 Microsoft
go.sum,github.com/konsorten/go-windows-terminal-sequences,MIT,2017 marvin + konsorten GmbH
go.sum,github.com/sirupsen/logrus,MIT,2014 Simon Eskildsen
```

File: LICENSE-3rdparty.csv [hunk 2]
```diff
@@ -54,3 +54,7 @@ go.sum,github.com/chzyer/readline,MIT,2015 Chzyer
 go.sum,github.com/chzyer/test,MIT,2016 Chzyer
 go.sum,github.com/google/pprof,Apache-2.0,Google Inc.
 go.sum,github.com/ianlancetaylor/demangle,BSD-3-Clause,2015 The Go Authors
+
```
Comment (jirikuncar) [inline LICENSE-3rdparty.csv:57]:
```suggestion
```

**Pull request merged**